tracer/src/Datadog.Trace/Debugger/ExceptionAutoInstrumentation/ExceptionTrackManager.cs

```diff
@@ -605,5 +615,10 @@ private static bool ShouldSkip(MethodBase method)
                 return true;
             }
         }
+
+        private static void SetDiagnosticTag(Span span, ExceptionReplayDiagnosticTag tag)
+        {
+            span.Tags.SetTag("_dd.di.er", tag.ToString());
```

Comment:
Given the heavy usage of Linq you probably don't care about individual allocations, but in case you do I recommend using Andrew's enum source generator thingy

Reply:
Implemented a workaround in e0c774b147415c446df132f754aec4a3606d1791 to minimize allocations. The code paths that lead to this method _could_ be hot.